domains/paimon.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "kokonut27",
+    "email": "beol0127@gmail.com",
+  },
+  "record": {
+    "CNAME": "paimon-is-a-dev.vercel.app"
```

Comment:
We don't currently support Vercel/Netlify. The current workaround is to use [Railway](https://railway.app/?referralCode=jj)

Reply:
Got it, I'll implement this ASAP.